Source/EasyNetQ/IAdvancedBus.cs

```diff
@@ -18,12 +17,12 @@ public interface IAdvancedBus : IDisposable
         /// <summary>
         /// True if a connection of the given type is established. 
         /// </summary>
-        bool IsConnected(PersistentConnectionType type);
+        bool IsConnected { get; }
 
         /// <summary>
         /// Establish a connection of the given type.
         /// </summary>
-        void Connect(PersistentConnectionType type);
+        void Connect();
```

Comment:
can't we make this async?

Reply:
Sure, let's do it. Just not to change this again in the nearest future.